src/main/java/frc/robot/HardwareInterface.java

```diff
@@ -0,0 +1,41 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj.PWMTalonSRX;
+
+/**
+ * Add your docs here.
+ */
+public class HardwareInterface {
+  private PWMTalonSRX motor1 = new PWMTalonSRX(1);
+  private PWMTalonSRX motor2 = new PWMTalonSRX(2);
+  private PWMTalonSRX motor3 = new PWMTalonSRX(3);
+  private PWMTalonSRX motor4 = new PWMTalonSRX(4);
+
+  /**
+   * Sets motor motorName to motorValue.
+   * @param motorName Name of the motor.
+   * @param motorValue Value to set the motor to.
+   */
+  public void setMotor(String motorName, double motorValue) {
```

Comment:
This might be a temporary solution as doing ifs like this doesn't scale well. For now it is good